css/src/components/message.scss

```diff
@@ -0,0 +1,52 @@
+.message {
+	display: grid;
+	grid-template:
+		'. time'
+		'avatar main';
+	grid-template-columns: max-content 1fr;
+	gap: $layout-1;
+
+	.message-time {
+		grid-area: time;
+		font-size: $font-size-9;
+	}
+
+	.message-avatar {
+		grid-area: avatar;
+		margin-block-start: $spacer-3;
+	}
+
+	.message-content {
+		grid-area: main;
+		padding: $spacer-5;
+		border: $border-width solid $border;
+		border-radius: $border-radius-lg;
+		box-shadow: $box-shadow-light;
+
+		.message-content-head {
+			display: grid;
+			grid-template-columns: 1fr auto;
+			margin-block-end: $spacer-2;
+
+			.message-content-display-name {
+				font-size: $font-size-7;
+				font-weight: $weight-semibold;
+			}
+
+			.message-content-options {
+				justify-self: flex-end;
+				margin-inline-end: $spacer-3;
+			}
+		}
+	}
+
+	&.message-right {
+		justify-self: flex-end;
+		width: 85%;
```

Comment:
Just wondering out loud, but would it be better to call this `message-indented` `message-sender` is quite awkward sounding, but that also might be more accurate. Feel free to resolve

Reply:
yea i was unsure, was better than what i put first which was .mine :D